m2cgen/assemblers/svm.py

```diff
@@ -57,6 +46,53 @@ def _assemble_single_output(self):
             ast.NumVal(intercept),
             *kernel_weight_mul_ops)
 
+    def _apply_kernel(self, support_vectors, to_reuse=False):
+        kernel_exprs = []
+        for v in support_vectors:
+            kernel = self._kernel_fun(v)
+            kernel_exprs.append(ast.SubroutineExpr(kernel, to_reuse=to_reuse))
+        return kernel_exprs
+
+    def _get_supported_kernels(self):
+        return {}
+
+    def _get_gamma(self):
+        raise NotImplementedError
+
+    def _get_output_size(self):
+        raise NotImplementedError
+
+    def _assemble_multi_class_output(self):
+        raise NotImplementedError
+
+    def _get_single_coef(self):
+        raise NotImplementedError
+
+    def _get_single_intercept(self):
+        raise NotImplementedError
+
+
+class SklearnSVMModelAssembler(BaseSVMModelAssembler):
+
+    def _get_supported_kernels(self):
+        return {
+            "rbf": self._rbf_kernel,
```

Comment:
Since all 4 of these kernels are shared between implementations and reside in the base class does it make sense to move the body of this `_get_supported_kernels` into the base class and make it a default behavior?

Reply:
Yeah, as we have only 2 implementations of SVM for now, I think it makes sense.